community/samples/serving/helloworld-java-quarkus/scripts/buildah.sh

```diff
@@ -0,0 +1,55 @@
+#!/bin/bash 
+
+# WORK_DIR  the directory where the application binaries are built
+# DESTINATION_NAME  - the fully qualified destination image name where the 
+# MVN_CMD_ARGS - the maven command arguments e.g. clean install
+# build image will deployed e.g. quay.io/myrepo/app:1.0
+
+set -eu
+
+PUSH=${PUSH:-'true'}
+
+cd $WORK_DIR
+
+ARTIFACT_NAME=$(mvn org.apache.maven.plugins:maven-help-plugin:3.1.1:evaluate -Dexpression=project.build.finalName -q -DforceStdout)
+ARTIFACT_NAME_PKG=$(mvn org.apache.maven.plugins:maven-help-plugin:3.1.1:evaluate -Dexpression=project.packaging -q -DforceStdout)
+
+APP_NAME="$ARTIFACT_NAME-runner"
+
+# build the java project 
+mvn ${MVN_CMD_ARGS:-clean -DskipTests install -Pnative}
+
+# define the container base image
+containerID=$(buildah from registry.fedoraproject.org/fedora-minimal)
+
+# mount the container root FS
+appFS=$(buildah mount $containerID)
+
+# make the native app directory
+mkdir -p $appFS/deployment
+
+cp target/$APP_NAME  $appFS/deployment/application
+
+chmod +x $appFS/deployment/application
+
+# Add entry  point for the application
+buildah config --entrypoint '["/deployment/application"]'  $containerID
+buildah config --cmd "\-Dquarkus.http.host=0.0.0.0" $containerID
+
+buildah config --author "devx@redhat.com" --created-by "devx@redhat.com" --label Built-By=buildah $containerID
```

Comment:
Should these @redhat.com email addresses be kept?

Reply:
this script is no longer needed, removed it